lib/rubocop/cop/style/block_delimiters.rb

```diff
@@ -342,16 +342,22 @@ def with_block?(node)
           node.respond_to?(:block_node) && node.block_node
         end
 
-        def get_blocks(node, &block)
+        def get_blocks(node, &block) # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
`Metrics/CyclomaticComplexity` may be resolved at some point in the future.  e.g.
https://github.com/rubocop/rubocop/commit/ac98713904b3e6f10605f00786e85bc3b3d0aa15

To avoid polluting `git blame` when that happens, it would be better to place annotations before and after the method, rather than inline.

```ruby
# rubocop:disable Metrics/CyclomaticComplexity
def get_blocks(node, &block)
  # impl...
end
# rubocop:enable Metrics/CyclomaticComplexity
```

Reply:
Updated.